eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -475,4 +475,5 @@ jobs:
     - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
         # ComponentGovernance is currently unable to run on pull requests of public projects. Running on non-PR
         # builds should be sufficient.
+        condition: ne(variables['Build.Reason'], 'PullRequest')
```

Comment:
Please add the first part of this condition https://github.com/Azure/azure-sdk-for-net/blob/master/eng/pipelines/templates/jobs/archetype-sdk-client.yml#L94.

Reply:
Will do, what does it do?

Reply:
exactly it will not hang up builds that we cancel.